gpt_index/tools/migrate_v1_to_v2.py

```diff
@@ -184,6 +187,8 @@ def index_dict_to_v2(struct: IndexDict) -> Tuple[V2IndexDict, List[V2Node]]:
         struct_v2 = V2WeaviateIndexDict(**dataclasses.asdict(struct_v2))
     if isinstance(struct, QdrantIndexDict):
         struct_v2 = V2QdrantIndexDict(**dataclasses.asdict(struct_v2))
+    if isinstance(struct, MilvusIndexDict):
+        struct_v2 = V2MilvusIndexDict(**dataclasses.asdict(struct_v2))
```

Comment:
nit: as noted above, since this script is only meant to migrate legacy serialized jsons, I don't think we need to add support for Milvus here.

Reply:
removed